configs/recognition/slowfast/slowfast_r50_8x8x1_256e_kinetics400_rgb.py

```diff
@@ -5,6 +5,6 @@
         resample_rate=4,  # tau
         speed_ratio=4,  # alpha
         channel_ratio=8,  # beta_inv
-        slow_pathway=dict(fusion_kernel=7)))
+        slow_pathway=dict(fusion_kernel=5)))
```

Comment:
May Need to add `fusion_kernel=7` for 8x8 in the future.

refer: https://github.com/facebookresearch/SlowFast/blob/master/configs/Kinetics/SLOWFAST_8x8_R50.yaml#L19